src/setupTests.js

```diff
@@ -0,0 +1,3 @@
+afterEach(() => {
+  jest.clearAllMocks()
```

Comment:
Did you see this? 

I don't understand enough how (and why!) the mocks behave when defined a) before describe blocks or b) in `beforeEach` blocks but if they are cleared after each test, it should at least make the situation better... 🤔 

This way all the mocks need to be re-defined in each test and tests shouldn't distract each other... (at least according to my best understanding...)